### DISCUSSION TITLE
NVidia graphic card overheating

### DISCUSSION THREAD

**Discussion by PonomarovP** [lllyasviel/Fooocus#2993] [Q&A] [answered]
Fooocus version 2.3.1
Graphic card: Geforce RTX 4070Ti
Driver version: 555.85 (latest)
When application sends model to GPU, it heats way to fast for fans to react, this leads to short-term (about 5 seconds) heating of the graphic card to 100-120 degrees each time the image is generated, which is not healthy and could damage the graphic card.

Is this a problem with fast graphic cards or I'm missing some configuration? I don't mind the process to be a bit slower if it rescues my GPU

**Comment by mashb1t** [comment, marked as answer]
This is not an issue with Fooocus but your graphics card and/or the fan mounted on it. Please ensure you have sufficient cooling capacity for your PC and especially GPU. You can also manually set the GPU fan to 100% (full speed) in your GPU driver or using a GPU manufacturer/vendor software.
